src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressMessage.cs

```diff
@@ -34,13 +32,20 @@ public SuppressMessage(string id, string target, string syntaxNode)
 			Id = id;
 			Target = target;
 			SyntaxNode = syntaxNode;
-
-			_hashCode = $"{Id}_{Target}_{SyntaxNode}".GetHashCode();
 		}
 
 		public override int GetHashCode()
 		{
-			return _hashCode;
+			int hash = 17;
```

Comment:
@metallmonster my idea was to calculate hashcode based on this algorithm once and store it like you did before in `_hashcode` private field. I just noted that the optimisation could be more effective without the creation of concatenated string during `_hashcode` calculation in the constructor if you calculate it with this method. 
However, it's different to say something about little optimisations without benchmarks so I think it's ok if you want to leave this variant.

Reply:
@SENya1990 I'll bring back `_hashCode` field